components/movementsensor/wheeledodometry/wheeled_odometry_test.go

```diff
@@ -0,0 +1,73 @@
+// Package wheeledodometry implements an odometery estimate from an encoder wheeled base
+package wheeledodometry
+
+import (
+	"context"
+	"testing"
+
+	"go.viam.com/rdk/components/motor"
+	"go.viam.com/rdk/testutils/inject"
+)
+
+// var count = 0
+
+func createFakeLeftMotor() motor.Motor {
```

Comment:
collapse into one function and return 1 or -1 based on the argument passed in (left or right)